data/tyzbit/update.info

```diff
@@ -0,0 +1 @@
+https://raw.githubusercontent.com/StevenBlack/hosts/master/data/StevenBlack/hosts
```

Comment:
Hi @tyzbit  the update.info file should point to the source of this list so future updates can be incorporated.

Thus URL above ^^^^ isn't correct.   Right?